QtlTodoList/Screen/ImageTodo/Camera/LaunchCameraView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  LaunchCameraView.swift
+//  QtlTodoList
+//
+//  Created by 柿崎逸 on 2023/08/19.
+//
+
+import SwiftUI
+
+/// カメラ起動
+struct LaunchCameraView: UIViewControllerRepresentable {
+    // MARK: - Property Wrappers
+    @Binding var image: UIImage
+    @Binding var isLaunchCameraView: Bool
+
+    class Coordinator: NSObject, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
```

Comment:
```suggestion
    class Coordinator: NSObject {}
    // MARK: - UIImagePickerControllerDelegate
    class Coordinator: UIImagePickerControllerDelegate {}
    // MARK: - UINavigationControllerDelegate
    class Coordinator: UINavigationControllerDelegate {}
```


分けておくとわかりやすいです

Reply:
こちらやってみたら以下のエラーが出ました。
`Invalid redeclaration of 'クラス名'`
つまり、同じ名前のクラスは２個以上作れないと怒られます…

Reply:
あ、これコードミスってるな

下二つはextensionだね

```swift
    class Coordinator: NSObject {}
    // MARK: - UIImagePickerControllerDelegate
    extension Coordinator: UIImagePickerControllerDelegate {}
    // MARK: - UINavigationControllerDelegate
    extension Coordinator: UINavigationControllerDelegate {}
```

Reply:
```suggestion
    class Coordinator: NSObject {}
    // MARK: - UIImagePickerControllerDelegate
    extension Coordinator: UIImagePickerControllerDelegate {}
    // MARK: - UINavigationControllerDelegate
    extension Coordinator: UINavigationControllerDelegate {}
```
上記のように２個目以降は拡張する形だとビルドは通る感じです！

Reply:
ありがとうございます！

Reply:
cdb2d21eefe9370d78dc18208022d72a71440cd7
修正しました！